### PR TITLE
Update languages to match osu-web 

### PR DESCRIPTION
**Pull request by jtbiddle**
Updated Language.cs to match the list of languages in osu! web (at least the ones I found in [osu-web/resources/lang](https://github.com/ppy/osu-web/tree/master/resources/lang))
Added Catalan, Persian, Filipino, Hebrew, Croatian, Lithuanian, Latvian, Malay, Slovenian, Serbian, Tajik, and Sinhala. 

Requested by peppy on Discord (https://discord.com/channels/188630481301012481/188630652340404224/1113739668358443038)

### CODE REVIEW

**Comment by jtbiddle** [comment]
Sorry, I opened this a bit prematurely (note to self: even if an issue sounds easy, test it). fa-IR, he, and si-LK don't work (show up as ??????) because of missing character sets or no right-to-left support and should be commented out.  fil, ms-MY and tg-TJ don't appear to have translations. The rest appear to function correctly.

File: osu.Game/Localisation/Language.cs [hunk 1]
```diff
@@ -105,6 +141,9 @@ public enum Language
         [Description(@"简体中文")]
         zh,
 
+        [Description(@"繁體中文")]
+        zh_tw,
+
```
Comment (cdwcgt) [inline osu.Game/Localisation/Language.cs:146]: Duplicate with "繁體中文（台灣）" (zh-hant)
I remember that dotnet does not support zh-tw
see https://github.com/ppy/osu-localisation-analyser/issues/44

**Comment by bdach** [comment]
> note to self: even if an issue sounds easy, test it

That is generally how we operate, yes. All code submitted should be tested. Preferably in an automated fashion, manually if infeasible.

> fil, ms-MY and tg-TJ don't appear to have translations.

Filipino should absolutely be working, but it doesn't seem to. Looks like the appropriate satellite assemblies aren't getting copied from the resources package. Malay does have translations and works. Tajik has translations on crowdin, but they're not in `osu-resources` yet, so that language is excused for now until proven otherwise.

**Comment by bdach** [comment]
I've pushed disables for languages that are currently broken to get this into a mergeable state. While I said Tajik has no localisation yet, in testing on a Windows 8.1 VM it does not work and raises a notification about `tg-TJ` [not being supported](https://github.com/ppy/osu/blob/51149343fba8f38cfe6a4954bec324cbe8234aa6/osu.Game/OsuGame.cs#L791) - for that language it wants `tg-Cyrl-TJ`, which means that we're going to have to do the same thingamajig we've done for `zh-{Hans,Hant}` for that language too.

I have no idea how to salvage Filipino; even installing the system language pack does not help.

Up to @ppy/team-client as to whether this is worth merging in the current state, or whether we want to fix the aforementioned issues first where reasonably feasible.

**Review by peppy — APPROVED** [review]

**Pull request merged**